retro-webamp-block.php

```diff
@@ -4,7 +4,7 @@
  * Plugin URI:        https://wordpress.org/plugins/retro-winamp-block/
  * Description:       A Winamp-styled audio block for all your retro music player needs.
  * Version:           1.0.1
- * Requires at least: 5.2
+ * Requires at least: 5.8
```

Comment:
The version should also be bumped in readme files.

https://github.com/10up/retro-winamp-block/blob/2d613feef8a03829524c6bef7f72b89f2334d970/readme.txt#L4

https://github.com/10up/retro-winamp-block/blob/2d613feef8a03829524c6bef7f72b89f2334d970/README.md?plain=1#L18